src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ManagedSslBio.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        internal static class ManagedSslBio
+        {
+            private unsafe readonly static ReadDelegate s_readDelegate;
+            private unsafe readonly static WriteDelegate s_writeDelegate;
+
+            internal static SafeBioHandle CreateManagedSslBio() => Crypto.CreateManagedSslBio();
+            
+            unsafe static ManagedSslBio()
```

Comment:
re: `unsafe`... is the C# compiler requiring that because the Read/WriteDelegate have pointers in their signatures?